rpm/kubelet.spec

```diff
@@ -5,7 +5,7 @@
 
 Name: kubelet
 Version: %{KUBE_VERSION}
-Release: %{RPM_RELEASE}
+Release: %{RPM_RELEASE}.1
```

Comment:
I would just bump the RPM_RELEASE counter here to signal a packaging change not an actual software change. Same for the .0 below.